engine/src/main/java/org/terasology/rendering/backdrop/Skysphere.java

```diff
@@ -101,16 +101,30 @@ public float getSunPositionAngle() {
         return celSystem.getSunPosAngle();
     }
 
+    /**
+     * Get the strength of the light for the current day part, based on sun/moon visibility.
+     * @return the strength of the current light in [0;1] range.
+     */
     @Override
     public float getDaylight() {
-        float angle = (float) Math.toDegrees(TeraMath.clamp(Math.cos(getSunPositionAngle())));
-        float daylight = 1.0f;
+        float lightBasedOnTimeOfDay = 1.0f;
+        final float ambientLight = 0.2f; // not totally black when the sun goes down, but the moon is not yet up
+        double radSunAngle = getSunPositionAngle(); // expected [-PI;PI] for the day cycle to work correctly!
+
+        // if the degree of the sun/moon is less than "lessLightUnderAngle", then we gradually increase the light, instead of full intensity
+        final double lessLightUnderAngle = Math.sin(Math.toRadians(15.0));
```

Comment:
In a few lines later, I check the absolute value of the sine compared to this, so that means fullIntensityThresholdInRadians == fullDarknessThresholdInRadians for now. I will experiment if using different thresholds for the sun and the moon would make it more pleasing overall (sun sets slower, but it takes more time for the moon to get to full intensity?